packages/components/src/panel/body.js

```diff
@@ -51,16 +51,22 @@ export class PanelBody extends Component {
 							onClick={ this.toggle }
 							aria-expanded={ isOpened }
 						>
-							{ isOpened ?
-								<SVG className="components-panel__arrow" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-									<G><Path fill="none" d="M0,0h24v24H0V0z" /></G>
-									<G><Path d="M12,8l-6,6l1.41,1.41L12,10.83l4.59,4.58L18,14L12,8z" /></G>
-								</SVG> :
-								<SVG className="components-panel__arrow" width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-									<G><Path fill="none" d="M0,0h24v24H0V0z" /></G>
-									<G><Path d="M7.41,8.59L12,13.17l4.59-4.58L18,10l-6,6l-6-6L7.41,8.59z" /></G>
-								</SVG>
-							}
+							{ /*
+								This span is wrapping the icon so when `isOpened` is updated the
+								state of `aria-expanded` isn't re-announced to NVDA
+							*/ }
```

Comment:
Thanks @timwright12! Actually, it's the opposite case: Firefox + NVDA don't announce aria-expanded because the browser repaints the whole element. Not sure we should patch it though, as next NVDA mitigates the issue.

Reply:
@afercia I updated the comment in case we end up wanting to merge this in. I do agree with your statement in https://github.com/WordPress/gutenberg/issues/1387 that we probably shouldn't rely on alpha releases of software that may or may not be released by the 5.0 milestone. Especially when it's something so small and easy to patch like this.

So if we want to push this fix, it's here, if not we should close out all the open items related to it, so no one else jumps in - and certainly remove it from the 5.0 milestone list.